redis_views/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.2.0'
+__version__ = '0.2.1'
```

Comment:
Thanks for remembering to bump this. Should we add a line in the CHANGELOG as well?